docs/framework-integrations/angular.mdx

```diff
@@ -48,10 +48,13 @@ instance can be passed into components as a `props` prop.
 
 The following plugins are available as Angular component wrappers:
 
-- `<uppy-dashboard />` renders [`@uppy/dashboard`](/docs/dashboard)
-- `<uppy-drag-drop />` renders [`@uppy/drag-drop`](/docs/drag-drop)
-- `<uppy-progress-bar />` renders [`@uppy/progress-bar`](/docs/progress-bar)
-- `<uppy-status-bar />` renders [`@uppy/status-bar`](/docs/status-bar)
+- UppyAngularDashboardModule renders [`@uppy/dashboard`](/docs/dashboard)
+- UppyAngularDashboardModalModule renders [`@uppy/dashboard`](/docs/dashboard)
+  as a modal
+- UppyAngularProgressBarModule renders
+  [`@uppy/progress-bar`](/docs/progress-bar)
+- UppyAngularStatusBarModule renders [`@uppy/status-bar`](/docs/status-bar)
+- UppyAngularDragDropModule renders [`@uppy/drag-drop`](/docs/drag-drop)
```

Comment:
Let's make sure all non-English words are in `<code>` tags
```suggestion
- `UppyAngularDashboardModule` renders [`@uppy/dashboard`](/docs/dashboard)
- `UppyAngularDashboardModalModule` renders [`@uppy/dashboard`](/docs/dashboard)
  as a modal
- `UppyAngularProgressBarModule` renders
  [`@uppy/progress-bar`](/docs/progress-bar)
- `UppyAngularStatusBarModule` renders [`@uppy/status-bar`](/docs/status-bar)
- `UppyAngularDragDropModule` renders [`@uppy/drag-drop`](/docs/drag-drop)
```

Reply:
Note that we probably should still list the `<uppy-dashboard />` form as the selector is defined by us https://github.com/transloadit/uppy/blob/4f91a0d529f67abc64943c201e74b2ee42788e37/packages/@uppy/angular/projects/uppy/angular/src/lib/components/dashboard/dashboard.component.ts#L17